examples/lighting-app/silabs/src/LightingManager.cpp

```diff
@@ -65,22 +65,23 @@ CHIP_ERROR LightingManager::Init()
     }
 
     bool currentLedState;
+    app::DataModel::Nullable<uint8_t> brightness;
```

Comment:
Why this change? how do you use the Brightness/ currentLevel in the LightingManager without RGB led at the moment?

Reply:
`mCurrentLevel` is using for posting the event to  `InitiateAction `if there is a change in value.
```
        if (mCurrentLevel != *aValue)
        {
            mCurrentLevel    = *aValue;
            action_initiated = true;
            AppTask::GetAppTask().PostLightLevelActionRequest(aActor, aAction, aValue);
        }
```